checkstyle.xml

```diff
@@ -2,47 +2,219 @@
+			<property name="ignoreComments" value="false"/>
+		</module>
+		
+		<!-- Disallow package.* imports -->
+		<module name="AvoidStarImport"/>
+		
+		<!-- Ensure static final field name format. Default format: "^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$" -->
+		<module name="ConstantName"/>
+		
+		<!-- Default case in switch/case statement has to come last -->
+		<module name="DefaultComesLast"/>
+		
+		<!-- Disallow "public boolean equals(Foo obj)" since this does not always override "public boolean equals(Object obj)" -->
+		<module name="CovariantEquals"/>
+		
+		<!-- Disallow empty statements (just a single ";") -->
+		<module name="EmptyStatement"/>
+		
+		<!-- Ensure that classes without public constructor are final -->
+		<module name="FinalClass"/>
+		
+		<!-- Ensure proper whitespace usage around "<" and ">" for generic types -->
+		<module name="GenericWhitespace"/>
+		
+		<!-- Ensure that classes with only static methods do not have a public constructor -->
+		<module name="HideUtilityClassConstructor"/>
+		
+		<!-- Disallow native literals -->
+		<module name="IllegalToken">
+			<property name="tokens" value="LITERAL_NATIVE"/>
+		</module>
+		
+		<!-- Ensure usage of "{" at the end of a line (and not on a newline) -->
+		<module name="LeftCurly"/>
+		
+		<!-- Ensure local final variable name format. Default format: "^[a-z][a-zA-Z0-9]*$" -->
+		<module name="LocalFinalVariableName"/>
+		
+		<!-- Ensure local non-final variable name format, except single-character variables in for loops. Default format: "^[a-z][a-zA-Z0-9]*$". -->
+		<module name="LocalVariableName">
+			<property name="allowOneCharVarInForLoop" value="true"/>
+		</module>
+		
+		<!-- Ensure non-static instance name format. Default format: "^[a-z][a-zA-Z0-9]*$" -->
+		<module name="MemberName"/>
+		
+		<!-- Ensure method name format and disallow methods to be named the same as the class (and therefore constructor). Default format: "^[a-z][a-zA-Z0-9]*$" -->
+		<module name="MethodName"/>
+		
+		<!-- Check method parameter padding -->
+		<module name="MethodParamPad"/>
+		
+		<!-- Check method type name format. Default format: "^[A-Z]$" -->
+		<module name="MethodTypeParameterName"/>
+		
+		<!-- Ensure that non-abstract classes define a constructor and not rely on the default one -->
+		<module name="MissingCtor"/>
+		
+		<!-- Ensure that both the @Deprecated and the @deprecated javadoc tag are present when one is present -->
+		<module name="MissingDeprecated"/>
+		
+		<!-- Ensure that the @Override tag is present when the {@inheritDoc} javadoc tag is present -->
+		<module name="MissingOverride"/>
+		
+		<!-- Ensure modifier order is 'first > last': public > protected > private > abstract > default > static > final > transient > volatile > synchronized > native > strictfp -->
+		<module name="ModifierOrder"/>
+		
+		<!-- Ensure that each variable has its own statement on its own line (disallows "i = j = k = 1;") -->
+		<module name="ModifierOrder"/>
```

Comment:
You have ModifierOrder twice here? Certainly that is not correct.

Reply:
This is indeed not correct, the second one should have been "MultipleVariableDeclarations".